apps/cookbook/src/app/examples/modal-example/modal-example-configuration/modal-example-configuration.component.html

```diff
@@ -51,4 +51,9 @@
     ></kirby-checkbox
     >Delay load additional modal content</label
   >
+  <label
+    *ngIf="openFullHeight !== undefined"
+    (click)="toggleOpenFullHeight((openFullHeight = !openFullHeight))"
```

Comment:
er det ikke dobbeltkonfekt med paranteser her?

Reply:
Yes, but it's not me, it's Prettier ;)